app/services/foreman_templates/template_importer.rb

```diff
@@ -25,6 +25,21 @@ def initialize(args = {})
     end
 
     def import!
+      if @repo.start_with?('http://', 'https://', 'git://')
+        import_from_git
+      elsif repo.start_with?('/')
```

Comment:
is it a good idea? I'd like to trigger sync from Foreman UI later, what would pwd be in case of foreman user? What if the task is run from cron? I think absolute paths is not a big limitation and can save us from few headaches :-)